plugins/callback/foreman.py

```diff
@@ -262,7 +266,9 @@ def append_result(self, result, failed=False):
         host = result._host.get_name()
         value = result._result
         value['failed'] = failed
+        check_mode = result._task.check_mode
         self.items[host].append((name, value))
+        self.check_mode = '1' if check_mode else '0'
```

Comment:
Interleaving the code here is kind of obfuscating.
```suggestion
        self.items[host].append((name, value))
        self.check_mode = '1' if result._task.check_mode else '0'
```

Reply:
I agree, changed it. Thanks :)